### DISCUSSION TITLE
Issue setting up nav

### DISCUSSION THREAD

**Discussion by ericthomas1** [mkdocs/mkdocs#3099] [Q&A] [answered]
Hello,

I have my `docs` directory setup like this:
![image](https://user-images.githubusercontent.com/42788729/211887675-8b2d1a8b-2db4-4454-af3e-ec75f3d54779.png)

I have the `nav:` section in the `mkdocs.yml` file setup like this:
![image](https://user-images.githubusercontent.com/42788729/211887831-3c24224b-2187-4182-914b-3e341a74ff4e.png)

The rendered webpage results in this where each doc directory is shown:
![image](https://user-images.githubusercontent.com/42788729/211887946-09e16240-c65d-48c5-8b8b-8190831795a8.png)

When user clicks a given directory, I'd like to see all the .md files stored in the directory as a nice table of contents that they can select from. Instead a get a 404 error :
![image](https://user-images.githubusercontent.com/42788729/211888107-e704a580-6ed4-4ec4-a941-885f9fa9a597.png)

What are the steps to make this happen?

Thank you

**Comment by oprypin** [comment, marked as answer]
MkDocs doesn't support a partially specified `nav`. Either all files are explicitly specified or none.
You could try simply dropping the `nav:` entirely.

**Reply by ericthomas1** [reply]
Thanks for the quick reply. 

Dropping the `nav:` altogether did the trick. I was making it way too complicated!

Thank you
